### PR TITLE
Start game automatically when spacebar is pressed

### PR DESCRIPTION
**Pull request by scooterbreak**
When you press spacebar on the countdown page the game will automatically begin

### CODE REVIEW

**Review by bricehalder — APPROVED** [review]
Looks good, would like to see a picture or video to check design

**Comment by scooterbreak** [comment]
Uploading Screen Recording 2023-11-18 at 10.13.58 PM.mov…

**Comment by scooterbreak** [comment]
https://github.com/wikispeedruns/wikipedia-speedruns/assets/64957454/bb850f56-b4fb-4011-9496-d48a2ca7e845

File: static/js/modules/game/countdown.js [hunk 1]
```diff
@@ -64,7 +76,7 @@ var CountdownTimer = {
             <slot></slot>
             <p>Good Luck!</p>
 
-            <div><button id="start-btn" class="btn btn-outline-secondary">(Don't want to wait? Start immediately!)</button></div>
+            <div><button id="start-btn" class="btn btn-outline-secondary">(Don't want to wait? Click here or press spacebar start immediately!)</button></div>
```
Comment (bricehalder) [inline static/js/modules/game/countdown.js:79]: This is getting kind of wordy. Maybe just change to `Click here or press spacebar to start immediately!`

**Review by bricehalder — CHANGES_REQUESTED** [review]

**Review by dqian3 — APPROVED** [review]
Looks good to me, just fix the typo and I'd be happy to merge :)

**Review by dqian3 — COMMENTED** [review]
A general note on deleting things:

Having a more systematic way to delete user data would be really nice, not just for this particular issue. We've also had people ask about deleting personal data, which we would have to do manually (and tbh I'm not sure if we've kept up with that). Going through and refactoring the SQL schema with CASCADE rules would help this along. This is a great first step though!

**Comment by mliu59** [comment]
> A general note on deleting things:
> 
> Having a more systematic way to delete user data would be really nice, not just for this particular issue. We've also had people ask about deleting personal data, which we would have to do manually (and tbh I'm not sure if we've kept up with that). Going through and refactoring the SQL schema with CASCADE rules would help this along. This is a great first step though!

Wrong PR? lol

**Comment by scooterbreak** [comment]
Edited the message. Thanks!

**Review by bricehalder — APPROVED** [review]

**Pull request merged**